src/lib/content/stats/author.js

```diff
@@ -9,10 +9,10 @@ function getAuthor()
   let author = null;
 
   // schema.org
-  let element = document.querySelector("[itemprop='author']");
+  let element = document.querySelector("[property='author' i],[itemprop='author' i]");
```

Comment:
Coding style: Make sure to run `npm run lint` (see also README) so that it can let you know about any coding style issues it finds. In this case, for instance, it'd point out that the line is longer than the 80 characters we defined.

Reply:
> Make sure to run `npm run lint` (see also README)

This [should be in CONTRIBUTING](https://github.com/blog/1184-contributing-guidelines). It would pop-up whenever someone opens a pull-request.

Reply:
You're right and it's already on our list of to-dos to finish open sourcing the extension (along with creating a CHANGES.md file and moving existing documentation over to the GitHub Wiki). So I'm sorry about the lack of those for now.

Reply:
I made an issue for this https://github.com/flattr/flattr-extension/issues/11

Reply:
circleci should automatically run our tests (including the linters) for pull requests now, so you won't have to run it locally if you don't want to, or forget to.  Once an update is pushed to this pull request it should run for this pr.